### PR TITLE
[ET-372]: WIP Automate package rename to delivery - **do not merge**

### PR DESCRIPTION
**Pull request by PrajaktaPurohit**
**Do not merge**

### CODE REVIEW

**Comment by schisamo** [comment]
Related mixlib-install PR: https://github.com/chef/mixlib-install/pull/176

File: app.rb [hunk 1]
```diff
@@ -296,14 +296,21 @@ def channel
...
   def project
-    # "automate" is not a published artifact, but is a valid product
-    # in mixlib-install where "automate" mimics the product metadata of "delivery".
-    # Until "automate" is published the omnitruck api needs to translate "automate" to "delivery".
+    # Chef Automate is published as an artifact 'automate' since version x.y.z
+    # In order to maintain backward conpatibility with it's previous name
+    # 'delivery' we swap the project name back to 'delivery' for versions
+    # older than x.y.z. Likewise if the request is for 'delivery' newer
+    # than x.y.z we swap the project name to 'automate.'
     if params['project'] == 'automate'
```
Comment (wrightp) [inline app.rb]: I see how you are trying to maintain backwards compat based on the existing hack. This was a situation where Chef announced `automate` as a product before putting the changes in place to support it. Let's take this opportunity to set the separation between `automate` and `delivery` artifacts.

If `automate` is specified then it should pass through. We shouldn't try to determine an old `delivery` version they intended. It should just not find a result.

Same for when `delivery` is passed. This should no longer be remapped now that properly versioned `automate` artifacts exist.

In essence, I think we should just remove the hack.

File: spec/app_spec.rb [hunk 2]
```diff
@@ -758,6 +758,74 @@ def app
       end
     end
 
+    context 'for automate' do
```
Comment (PrajaktaPurohit) [inline spec/app_spec.rb]: These tests still need some tweaking to test the actual conditional.

**Pull request closed without merging**